tests/Fixtures/packageJson/stricter_constraints_package.json

```
@@ -0,0 +1,11 @@
+{
+   "name": "symfony/fixture",
+   "devDependencies": {
+      "@hotcookies": "^2",
+      "@hotdogs": "2.4.1",
```

The `@hotdogs` constraint in the package is `^2`. So let's use the following, so that we can test that the constraint WILL be updated if it doesn't match (so we'll use `@hotcookies` to test that satisfying constraints are NOT changed and we'll use `@hotdogs` to test that non-satisfying ARE changed):

```suggestion
      "@hotdogs": "^1.9",
```